src/client/src/sockets.rs

```diff
@@ -119,8 +119,17 @@ impl ClientConfig {
     }
 
     pub(crate) async fn connect(self) -> Result<VersionedSocket, FluvioError> {
+        let addr = self.addr.clone();
+        // When we have websocket proxy in place, this won't be needed.
+        #[cfg(target_arch = "wasm32")]
+        let addr = if addr == "localhost:9010" {
+            String::from("ws://localhost:3001")
+        } else {
+            addr
+        };
```

Comment:
@sehz Thoughts on what to do in place of this?

Reply:
This can passed as addr.  There is no need for hard coding

Reply:
Yeah, so the reason this is a needed work around is that the client connects to fluvio-sc via a webscocket, then once trying to get a Partition Consumer stream or a producer, the SC gives some metadata about the SPU to connect to which is `localhost:9010` for a local cluster.

This isn't exposed to the client.

When we add a real websocket proxy rather than running `websocat -v --binary ws-l:127.0.0.1:3000 tcp:127.0.0.1:9003`, it needs to do something smarter.

Reply:
I removed it from the PR but I'm still using it in my local branch.